Languages/IronPython/IronPython/Compiler/Ast/DelStatement.cs

```diff
@@ -43,7 +43,7 @@ public class DelStatement : Statement {
                 statements.Add(_expressions[i].TransformDelete());
             }
             statements.Add(AstUtils.Empty());
-            return GlobalParent.AddDebugInfo(MSAst.Expression.Block(statements), Span);
+            return /*AppendLine*/(GlobalParent.AddDebugInfo(MSAst.Expression.Block(statements), Span));
```

Comment:
I will remove all of them as requested in a sec.
(I initially added the commented out function calls to problematic places - as a place holder, to tell that they are not supposed to be there and it's not a mistake.)